oncall/autoIssueAssign.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+autoAssign() {
+  curl -X POST "https://api.github.com/repos/microsoft/Oryx/issues/$3/assignees" -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $2" -d "{\"assignees\": ["\"$1\""]}"
+}
+
+checkCustomOncall() {
+  input=$1
+  today=$2
+  token=$3
+  issue=$4
+  currentOncall=$5
+  if [ -z "$input" ]; then
+    return 1
+  fi
+
+  IFS="," read -ra arr <<< "$1"
+  name=${arr[0]}
+  startDate=${arr[1]}
+  endDate=${arr[2]}
+  startDate=$(date -d $startDate +%s)
+  endDate=$(date -d $endDate +%s)
+  if [ "$today" -ge "$startDate" ] && [ "$today" -le "$endDate" ]; then
+    echo "Custom shift found"
+    autoAssign $name $token $issue
+    return 0
+  else
+    autoAssign $currentOncall $token $issue
+    return 0
+  fi
```

Comment:
I think we should handle the case where multiple entries are provided in the custom on-call rotation list -- currently we're just checking the first line of the Variable.

I would suggest following something like the following pseudocode:

```
assignCustomOncall()
  customRotation = parseVariable()
  for each entry in customRotation
    if entry is in dateRange
      assignIssue()
      return
```